NBitcoin/Scripting/Parser/StringInput.cs

```diff
@@ -31,12 +32,12 @@ public IInput<char> Advance()
 
 		public IEnumerator<char> GetEnumerator()
 		{
-			return this.Source.GetEnumerator();
+			return this.Source.AsEnumerable().GetEnumerator();
```

Comment:
?? should not be needed

Reply:
I thought so too, but build fails when `netstandard1.3`

Reply:
Weird ... with `AsEnumerable()` , netstandard1.3 build succeeds in local. But not in CI.

Reply:
I have to go a sleep. Will think about this tomorrow. (I may just give up supporting IEnumerable<T> for IInput and rollback to the original.)

Reply:
Fixed.